docs/docs/20-usage/35-advanced-yaml-syntax.md

```diff
@@ -0,0 +1,62 @@
+# Advanced YAML syntax
+
+## Anchors & aliases
+
+You can use [YAML anchors & aliases](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) as variables in your pipeline config.
+
+To convert this:
+```yml
+pipeline:
+  test:
+    image: golang:1.18
+    command: go test ./...
+  build:
+    image: golang:1.18
+    command: build
+```
+
+Just add a new section called **variables** like this:
+
+```diff
++variables:
++  - &golang_image 'golang:1.18'
+
+ pipeline:
+   test:
+-    image: golang:1.18
++    image: *golang_image
+     command: go test ./...
+   build:
+-    image: golang:1.18
++    image: *golang_image
+     command: build
+```
+
+## Example of YAML override and extension
```

Comment:
@6543 shouldn't this second paragraph also be displayed [here](https://woodpecker-ci.org/docs/next/usage/advanced-yaml-syntax)? I only see the first paragraph. But maybe I misunderstood the purpose of the "next documentation".

Reply:
@svdHero it's not working so it go hiden until fixed